packages/client/src/components/Button/Button.tsx

```diff
@@ -42,6 +46,7 @@ export const Button = ({
     basicButton: true,
     subButton: view === 'subButton',
     'basicButton--link': view === 'link',
+    'button--loading': loading,
```

Comment:
По БЭМ-у правильно называется так:
```suggestion
    'basicButton_link': view === 'link',
    'button_loading': loading,
```
Насколько я понял, они у вас [модификаторы](https://ru.bem.info/methodology/quick-start/#%D0%BC%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80).